### ISSUE TITLE
Initialize student model's weights

### ISSUE THREAD

**Issue by duytue-kite** [OpenGVLab/VideoMAEv2#46]
Hello! Thank you for the great work!

In the paper, page 11, there is a sentence:
```
Distillation results.
Using the procedure of [9], we are able to compress VideoMAE V2-g into a mush smaller ViT-B. Specifically, we initialize the student model with the VideoMAE V2-B weights after the post-pre-training.
```
Can you explain this process? Where is this part in the code? How can I do such initialization?

My intention is to just use a ViT-Small model for finetuning by loading from a bigger pretrained model like vit_base or vit_huge.
I tried setting `--model vit_small_patch16_224` while loading `MODEL_PATH=vit_g_hybrid_pt_1200e.pth` but it didn't load weights due to size mismatching.
```
MODEL_PATH='YOUR_PATH/model_zoo/vit_g_hybrid_pt_1200e.pth'  # Model for initializing parameters
...
--model vit_giant_patch14_224 \
...
```

Thank you!

**Comment by congee524** [comment]
We don't release the codes about distillation. The distillation following [9] is to align the output (logits) of the student model (vit-s/vit-b) with that of the teacher model (vit-g) during finetuning on the K710 dataset.

It looks like what you're trying to do is directly load the parameters from the big model onto the small model. Indeed some work has found it works (I'm not sure), but then you should manually remove some of the parameters from the large model so that the shape of the model weights is as large as the small model's.

**Comment by congee524** [comment]
> So the method mentioned in the paper is about training student + teacher model in a knowledge distillation fashion instead of just initialize the student model with the teacher's weights.

Exactly~

> I'm trying to train a model for Spatio-temporal action localization (custom dataset based on AVA). However, I only have limited GPU resources (4 GPUs). I was wondering if you might have any suggestions or advice on training a smaller model like ViT-small for VideoMAEv2.

If you only have 4 gpu's, you can try to refine the features extracted by videomae v2 with an adapter (freeze videomae v2), or use some special fine-tuning method like LoRA.
